Sources/ApexyURLSession/URLSessionClient.swift

```diff
@@ -0,0 +1,121 @@
+import Foundation
+import Apexy
+
+final public class URLSessionClient: Client {
```

Comment:
Should we make it `open class` to let developers to add some functionality?

Reply:
Yes, it’s a good idea to make it `open`.